keyless/circuit-data/templates/helpers/misc.circom

```diff
@@ -63,15 +63,19 @@ template JWTFieldCheck(maxKVPairLen, maxNameLen, maxValueLen) {
 }
 
 // Checks if character 'char' is a whitespace character. Returns 1 if so, 0 otherwise
+// Assumes char is a valid ascii character. Does not check for non-ascii unicode whitespace chars.
 template isWhitespace() {
    signal input char;  
                        
    signal is_tab <== IsEqual()([char, 9]); // character is a tab space
-   signal is_newline <== IsEqual()([char, 10]); // character is a newline 
-   signal is_carriage_return <== IsEqual()([char, 13]); // character is a carriage return
+
+   signal is_line_break_part_1 <== GreaterEqThan(8)([char, 10]); // character is a newline 
+   signal is_line_break_part_2 <== LessEqThan(8)([char, 13]); // character is a carriage return
+   signal is_line_break <== is_line_break_part_1 * is_line_break_part_2;
```

Comment:
Please change to a logical OR subcircuit for readability

Reply:
https://github.com/iden3/circomlib/blob/master/circuits/gates.circom#L37

Reply:
This is a AND not an OR?

Reply:
also by that logic shouldn't we change the last line to use the OR template rather than just using addition? I used the raw operator here to be consistent with that line

Reply:
You're right. Can you change the comments to indicate the lines cover a range of characters now, not just newline and carriage return?

Reply:
I think we should change the last line to a MultiAND. It is also in the circom standard library

Reply:
Looking at the code for MultiAND I'm not sure what it does...

Reply:
<img width="670" alt="image" src="https://github.com/aptos-labs/aptos-core/assets/103289/e3660420-ae5b-4319-ba30-62d8aa095dbd">

for context in case we forget later